qa/tasks/cephfs/filesystem.py

```diff
@@ -272,8 +272,14 @@ def get_nonnumeric_values(value):
                      "-Infinity": -float("inf")}
                 return c[value]
 
-            j = json.loads(response_data.replace('inf', 'Infinity'),
-                           parse_constant=get_nonnumeric_values)
+            
+            j = "" 
```

Comment:
apologies for suggestion double quotes, `json.loads()` returns dict, therefore this should be `j = {}`

Reply:
Oh, yes. My bad